crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/FullAssessmentAvailabilityServiceTest.java

```diff
@@ -0,0 +1,157 @@
+package uk.gov.justice.laa.crime.meansassessment.service;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.justice.laa.crime.meansassessment.data.builder.TestModelDataBuilder;
+import uk.gov.justice.laa.crime.meansassessment.model.common.ApiCreateMeansAssessmentRequest;
+import uk.gov.justice.laa.crime.meansassessment.model.common.ApiCreateMeansAssessmentResponse;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CaseType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.NewWorkReason;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FullAssessmentAvailabilityServiceTest {
```

Comment:
I know it's trivial, but can we keep the naming of the tests consistent with the other classes by following the given_when_then naming convention.